### PR TITLE
remove unnecessary uses of object

### PR DESCRIPTION
**Pull request by mokibit**
Removed remaining unnecessary uses of object.

### CODE REVIEW

File: master/buildbot/changes/github.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 }
 
 
-class PullRequestMixin(object):
+class PullRequestMixin():
```
Comment (rodrigc) [inline master/buildbot/changes/github.py]: For classes which you changed to not inherit from anything, you should just remove the parentheses.

File: master/buildbot/test/fake/private_tempdir.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def cleanup(self):
         pass
 
 
-class MockPrivateTemporaryDirectory(object):
+class MockPrivateTemporaryDirectory():
```
Comment (rodrigc) [inline master/buildbot/test/fake/private_tempdir.py]: Remove parens

File: master/buildbot/test/unit/test_util_kubeclientservice.py [hunk 3]
```diff
@@ -40,7 +40,7 @@
 from buildbot.util import kubeclientservice
 
 
-class MockFileBase(object):
+class MockFileBase():
```
Comment (rodrigc) [inline master/buildbot/test/unit/test_util_kubeclientservice.py]: Same

File: master/buildbot/test/unit/test_worker_kubernetes.py [hunk 4]
```diff
@@ -27,7 +27,7 @@
 from buildbot.worker import kubernetes
 
 
-class FakeBot(object):
+class FakeBot():
```
Comment (rodrigc) [inline master/buildbot/test/unit/test_worker_kubernetes.py]: Same

File: master/buildbot/test/unit/test_worker_kubernetes.py [hunk 5]
```diff
@@ -40,7 +40,7 @@ def loseConnection(self):
         self.n()
 
 
-class FakeResult(object):
+class FakeResult():
```
Comment (rodrigc) [inline master/buildbot/test/unit/test_worker_kubernetes.py]: Same

File: master/buildbot/test/unit/test_worker_kubernetes.py (hunk 4 above)
Comment (mokibit) [reply master/buildbot/test/unit/test_worker_kubernetes.py]: Thanks, already fixed.

File: master/buildbot/util/service.py [hunk 6]
```diff
@@ -51,7 +51,7 @@ def reconfigServiceWithBuildbotConfig(self, new_config):
 
 # twisted 16's Service is now an new style class, better put everybody new style
```
Comment (tardyp) [inline master/buildbot/util/service.py]: this comment is obsolete

**Pull request merged**